Makefile

```diff
@@ -130,11 +130,11 @@ format:
 
 .PHONY: test
 test:
-	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
+	@SKIP_FETCH_TOOLS=1 $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
```

Comment:
what's the purpose of this change?

Reply:
It skips fetching the binaries needed for envtest integration tests (etcd, kube-apiserver, kubectl) (see https://github.com/kubernetes-sigs/controller-runtime/tree/master/pkg/envtest and https://github.com/gardener/gardener/blob/master/hack/setup-envtest.sh).
Currently, there are no envtest test here, I assume. So no need, to fetch those binaries.

Reply:
thanks for explanation.